### DISCUSSION TITLE
What export_options do you need top use in gym (or build_app) to get it ready for TestFlight?

### DISCUSSION THREAD

**Discussion by tomvisions** [fastlane/fastlane#17743] [General]
Hey Folks, I trying to to get my iOS (mobile - iPhone)app ready for TestFlight.
I've run fastlane match development and it runs cleanly

This is what my Fastlane looks like

default_platform(:ios)

platform :ios do
  desc "Push a new beta build to TestFlight"
  lane :beta do
    increment_build_number(xcodeproj: "name.xcodeproj")

    match(force: true, git_branch: "master", git_full_name: "name", git_user_email: "email", app_identifier: "org.app.test", type: "development", verbose: true, keychain_password: "test")
    gym(configuration: "Debug", workspace: "name.xcworkspace", scheme: "schema_name", export_method: "development")
    upload_to_testflight
  end
end

In the log, it gives me brief if I use development for export_method.

Your `export_method` in gym is defined as `development`
ERROR [2020-12-07 00:37:45.45]: which might cause problems when signing your application
ERROR [2020-12-07 00:37:45.45]: Are you sure want to build and export for development?
ERROR [2020-12-07 00:37:45.45]: Please make sure to define the correct export methods when calling
ERROR [2020-12-07 00:37:45.45]: gym in your Fastfile or from the command line

Not sure what export method I should use for building the app to get it ready for TestFlight.

Would appreicate some insight.

Tom

**Comment by tomvisions** [comment]
Figured it out.

**Reply by rogerluan** [reply]
Mind posting your solution, to possibly help future readers? 😊

**Reply by cloudworkpro-hassen** [reply]
instead of development, did you use ad-hoc?
